dev.neovide.neovide.yml

```diff
@@ -0,0 +1,18 @@
+id: dev.neovide.neovide
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: neovide
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host
+  - --share=network
+modules:
+  - modules/neovim.yml
+  - modules/host-spawn/host-spawn.yml
+  - modules/neovide/neovide.yml
```

Comment:
and just to reiterate, these non autogenerated portion of the manifest should be inline here like they were before.